betatest/admin.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015 OpenCraft <xavier@opencraft.com>
```

Comment:
All the section on how to apply the licence says is

> It is safest to attach them to the start of each source file to most effectively state the exclusion of warranty.

It doesn't say this is necessary for the licence to be valid or anything.

(And I wouldn't trust legal advice from the FSF too much.  It seems to be generally agreed that substantial parts of the AGPL are unenforcable under US law; at least that's what I heard from an FSF member.)